docs/design/2023-03-15-ticdc-storage-sink.md

```diff
@@ -53,19 +53,22 @@ This section describes the storage path structure of data change records, metada
 Using the csv protocol as an example, files containing row change events should be organized as follows:
 
 ```
-s3://bucket/prefix1/prefix2                 <prefix>
+s3://bucket/prefix1/prefix2                                   <prefix>
 ├── metadata
-└── schema1                                 <schema>
-    └── table1                              <table>
-        ├── 10000                           <table-version-separator>
-            ├── 13                          <partition-separator, optional>
-            │   ├── 2023-03-23              <date-separator, optional>
-            │   │   ├── CDC000001.csv
-            │   │   └── CDC.index
-            │   └── 2023-03-24              <date-separator, optional>
-            │       ├── CDC000001.csv
-            │       └── CDC.index
-            └── schema.json
+└── schema1                                                   <schema>
+    ├── meta
+    │   └── schema_441349361156227000_3233644819.json
```

Comment:
As an example , it is the best if  the  version under the table directory  is the same as the one in the schema json

Reply:
There is no corresponding data to write in the directory corresponding to the database schema file.